### ISSUE TITLE
Bug de la sanación de Sed de Sangre del Guerrero

### ISSUE THREAD

**Issue by jandrox** [RoDNet/Bugtracker-Infinium-4.0.6a#478]
Hola, tengo un problema que la sanación del golpe Sed de sangre (http://es.wowhead.com/spell=23881) no cura a mi parecer ni una mínima parte de lo que debería curar, y menos si le añades el glifo de Sanación sangrienta y el talento Vendaje de Batalla, como se puede ver en las capturas tan solo cura 582 de daño, cuando debería ser con 116.000 de vida el 
(1%(Sed de sangre)+ 6%(Vendaje de batalla))= 8120+ 3248(40% Glifo) ==  11.368 
según aparece en las habilidades a mi parecer.

![WoWScrnShot_012613_061246](https://f.cloud.github.com/assets/3144712/99513/fa9d1a94-6777-11e2-9e56-563e8d6736b1.jpg)
![WoWScrnShot_012613_061238](https://f.cloud.github.com/assets/3144712/99516/12e3d138-6778-11e2-9623-9c5d958d6bbf.jpg)
![WoWScrnShot_012613_061255](https://f.cloud.github.com/assets/3144712/99518/27c2d766-6778-11e2-91cb-65d0fe7dcd9d.jpg)

Un saludo.

**Comment by DatJetto** [comment]
Te has hecho un lío; sería así:
Tu vida 116.000 el 0.7 % = 812
40% de 812 = 324.8
6% de 812 = 48.72
Ahora sumamos todo 812 + 324.8 + 48.72 = Deberias curarte entorno a 1186 ( Dado que tu vida no es 116.000 justo curarías un poco más )

**Comment by jandrox** [comment]
Ah ok, pero de todas maneras sería el 20% de vendaje de batalla que me he confundido y es autosanación lo que daría 812 + 324.8 + 162.4 = 1299.2 que es más del doble de los 582 que me está curando no??

Un saludete.

**Comment by DatJetto** [comment]
Si tienes razón, no me había fijado en el 20% es. 
*Tienes algun talento que mejore Sed de sangre o que varíe el 0.7% de tu vida? Con mi guerrero me dice 0.5% ( es nivel 9 xD ) o mejora al subir al 85?  Es para probar la posible variación de este en un Glifo o talento :D

**Comment by jandrox** [comment]
Es el glifo de Sanación sangrienta el que te aumenta ese poquejo, pero viene muy bien ;)

**Comment by asfo** [comment]
Lo voy a probar ya que ahí se muestra sobre sanación, esto, puede afectar la curación real al entorpecer la cantidad e HP que debe sanar.

**Comment by DatJetto** [comment]
@jandrox Entonces sería:
812 + 162.4 = 976 aprox.

**Comment by jandrox** [comment]
Te has comido el 40% del glifo de sanación sangrienta ;)

**Comment by DatJetto** [comment]
No, porque el glifo es el 0.2% extra, añadí 0.7 ( el cual ya está añadido del glifo ) siendo base 0.5

## Como expuse antes 

Te has hecho un lío; sería así:
Tu vida 116.000 el 0.7 % = 812
40% de 812 = 324.8
6% de 812 = 48.72
Ahora sumamos todo 812 + 324.8 + 48.72 = Deberias curarte entorno a 1186 ( Dado que tu vida no es 116.000 justo curarías un poco más )

**Comment by jandrox** [comment]
Me acabo de dar cuenta que estamos los dos equivocados, siendo la base el 0.5% el 40% es el 0.2% lo cual da el 0.7% que es la base de curación con el glifo de Sanación sangrienta, de aquí se deduce que el juego hace los cálculos en base a porcentajes no a valores, y en base a esto sale claramente que no suma el 20% del talento Vendaje de batalla. 
Si hacemos los cálculos en los porcentajes la sanación recibida debe ser : 0.5(base) + 0.2(40% del glifo) + 0.14 (20% del talento) = 0.84 % de sanación.

Si realizamos los cálculos vemos que está usando el 0.5% sin añadir ni talento ni glifo, ya que el 0.5% de 116400 es 582 como podemos observar en las capturas de pantalla.

Con esto se ven 3 bugs:

1º No añade el 20% del talento Vendaje de batalla
2º No añade el 40% del glifo de Sanación sangrienta.
3º Bug visual ya que aunque aparece como el 0.7 no es real.

Un saludo.
